src/Common/src/Internal/Runtime/MetadataBlob.cs

```diff
@@ -22,7 +22,7 @@ internal enum ReflectionMapBlob
         DefaultConstructorMap                       = 14,
         UnboxingAndInstantiatingStubMap             = 15,
         InvokeInstantiations                        = 16, // unused
-        PointerTypeMap                              = 17, // unused
```

Comment:
I'm not quite sure whether to reuse this unused one or define a new value for DelegateMarshallingStubMap?